src/finder/checks/AuxclickJSCheck.js

```diff
@@ -7,14 +7,21 @@ export default class AuxclickJSCheck {
     this.type = sourceTypes.JAVASCRIPT;
   }
 
-  match(astNode, astHelper){
+  match(astNode, astHelper, scope) {
     if (astNode.type !== 'NewExpression') return null;
     if (astNode.callee.name !== 'BrowserWindow') return null;
 
     let location = [];
 
     if (astNode.arguments.length > 0) {
-      const found_nodes = astHelper.findNodeByType(astNode.arguments[0],
+      
+      var target = {};
```

Comment:
This is something I've been thinking a lot since my initial contributions to the checks. On the other hand I feel that the already existing `astHelper.findNodeByType` is providing us with a lot of control over the parsed ast tree, which may comes in handy at times. 
Were you thinking something like:
`findNodes(targetAstTree, arrayOfNodeNamesOrValuesToLookFor, callbackIfNodeIsFound)`? Maybe we could file a new separate issue and discuss there the details.

Reply:
What I meant was not `findNodeByType`, but:
```cs
      var target = {};
      if (scope.resolveVarValue)
        target = scope.resolveVarValue(astNode);
      else
        target = astNode.arguments[0];
``` 
Looks like it is everywhere the same.

Reply:
Since scope parsing with [escope](https://github.com/estools/escope) is not yet supported in typescript I added that snippet until we get a working [eslint-scope](https://github.com/eslint/eslint-scope) implementation. By the way, I do agree that this piece of code in every check is redundant and I consequently added these stub functions in d6c348efda45900fb9f1def08654f76f736fc036 to keep things clean:
``` Javascript
    data.Scope = {}; // new Scope(data);
    data.Scope.resolveVarValue = (astNode) => astNode.arguments[0];
    data.Scope.updateFunctionScope = () => {};
```